frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,44 @@
+.errorMessage {
+  padding: 0;
+  margin: 0;
+  margin-bottom: var(--font-size-xxs);
```

Comment:
We're using the `margin` shorthand to reset all values to `0` and then we add `margin-bottom` shorty after.

Also, the default styles for a `p` element only have `margin-bottom` and `margin-top` applied so I wouldn't use a shorthand to reset those.

Reply:
I would also nudge us to think of a reset/normalize since we'll be dealing with a lot more `p` elements in the project.

Reply:
I disagree with you here @mirhamasala, why do you think it's better to reset them more verbosely than just using the shorthand?

Reply:
Something I learned and find useful, explained here: https://codeguide.co/#shorthand-notation

Reply:
And I think this commit kinda proves that point, right?

We had both:

```
margin: 0;
margin-bottom:  var(--font-size-xxs);
```

Reply:
The argument is that it leads to slopier code and unintended side effects, I don't see how that applies here, especially since you understood right away that it was supposed to be a margin reset. What side effects should we be expecting?

Reply:
I didn't understand straight away, actually–and that's the thing.

Reply:
Besides, if you ever need to add `right` and `left` spacing, this makes even less sense.

Reply:
I fail to see how `margin-top: 0` is supposed to be more explicit about the intention of the rule than `margin: 0` but maybe we should just disagree here.

Reply:
`margin: 0` is clear (although, again, unnecessary, because you only want to reset the `top`)

But `margin: 0` followed with `margin-bottom: 12px` is more confusing than `margin-top: 0` `margin-bottom: 12px`